.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
 max-line-length = 120
-ignore = E305,E402,E721,E741,F401,F403,F405,F999,W503,W504
-exclude = build,docs/source,_ext,third_party
+ignore = E305,E402,E721,E741,F401,F403,F405,W503,W504,F821,F841,F999
```

Comment:
What happens if `F821` and `F841` are not added here? They should not be added here I think.

Reply:
Can you also see what happens if removing 401, 402, 403, 405, 503, 504? I do not see a good reason in torchaudio that they should be ignored.

Reply:
ref https://github.com/pytorch/pytorch/blob/master/.flake8#L7

Reply:
I agree that we should remove `F821,  F841`

Reply:
`F503, F504` looks like its being ignore in pytorch too.

https://www.flake8rules.com/rules/W504.html
https://www.flake8rules.com/rules/W503.html

From what I understand they seem to be contradicting each other

Reply:
`403,405` being ignore by pytorch too. (Similar rules discouraging the use for * )


https://www.flake8rules.com/rules/F403.html
https://www.flake8rules.com/rules/F405.html

Reply:
![image](https://user-images.githubusercontent.com/4648995/104123306-8358a780-5370-11eb-94f6-ff7a3ac124e6.png)

 looks like we have `401` violations.

Reply:
Would it possible for you to fix it?

For `__init__.py`s, listing the imported stuff in `__all__` would do. https://stackoverflow.com/a/31079085
For others, unused import should be removed.

If that uncovers other issues, (like unused imported function is used from external module). We can just give up and do it in a follow up.